.depcheckrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "ignores": [
+    "clipanion",
+    "@lavamoat/allow-scripts",
+    "@lavamoat/preinstall-always-fail",
+    "@metamask/auto-changelog",
+    "@types/*",
+    "@yarnpkg/*",
```

Comment:
The addition of `clipanion` and `@yarnpkg/*` is different from the module template.

It seems that newer versions of `depcheck` (at the time of this writing, I was testing with 1.4.7) cause `depcheck` to mistakenly think that none of the packages being referred to in the `.yarn/plugins` directory are installed: 

```
Missing dependencies
* @yarnpkg/core: ./.yarn/plugins/@yarnpkg/plugin-constraints.cjs
* @yarnpkg/cli: ./.yarn/plugins/@yarnpkg/plugin-constraints.cjs
* clipanion: ./.yarn/plugins/@yarnpkg/plugin-constraints.cjs
* @yarnpkg/fslib: ./.yarn/plugins/@yarnpkg/plugin-constraints.cjs
```

We probably need to add these lines to the module template.